src/extension.ts

```diff
@@ -187,8 +187,15 @@ async function activateServerForFolder(context: ExtensionContext, uri: Uri, fold
 
   let serverExecutable;
   try {
+    let localServer: string | null;
+    if (workspace.getConfiguration('haskell').get('ignorePATH') === true) {
+      localServer = null;
+      logger.info('Ignoring haskell-language-server on PATH');
+    } else {
+      localServer = findLocalServer(context, logger, uri, folder);
+    }
```

Comment:
Shouldn't this logic happen in `findLocalServer`?

Maybe pass in an extra option?

Reply:
Thanks for your advice, revised by [8c02da9](https://github.com/haskell/vscode-haskell/pull/534/commits/8c02da90226ca7ae4d8cc6a809a2747c17dfed2e)